src/server/game/Battlegrounds/BattlegroundQueue.h

```diff
@@ -68,6 +70,8 @@ class BattlegroundQueue
         bool GetPlayerGroupInfoData(uint64 guid, GroupQueueInfo* ginfo);
         void PlayerInvitedToBGUpdateAverageWaitTime(GroupQueueInfo* ginfo);
         uint32 GetAverageQueueWaitTime(GroupQueueInfo* ginfo) const;
+        uint32 GetPlayesCountInGroupsQueue(BattlegroundBracketId bracketId, BattlegroundQueueGroupTypes bgqueue);
```

Comment:
GetPlayerCountInGroupsQueue?

Reply:
Yes

Reply:
Typo:
GetPlaye**s**CountInGroupsQueue
Should be renamed to
GetPlaye**r**CountInGroupsQueue